Sources/NIO/EventLoop.swift

```diff
@@ -116,14 +120,20 @@ public final class RepeatedTask {
                 self.reschedule0()
             }
         }
+
+        scheduled.futureResult.whenFailure { _ in
```

Comment:
Can you ignore the result but specify its type? That helps us catch unexpected API changes. E.g., write `{ (_: Error) in`